abs.yaml

```diff
@@ -0,0 +1,4 @@
+#upload_channels:
```

Comment:
please remove

Reply:
yes, waiting for approval first

Reply:
oh I see, we uncomment this to upload to snowflake and then delete it.